### PR TITLE
add lojban-dataset.scm

### PR DESCRIPTION
**Pull request by Bitseat**

### CODE REVIEW

File: tests/learning/xpattern-miner/c++-based/XPatternMinerUTest.cxxtest [hunk 1]
```diff
@@ -716,6 +717,66 @@ void XPatternMinerUTest::test_SodaDrinker()
...
+	// And
+	//   Inheritance
+	//     X
+	//     Concept "man"
+	//   Inheritance
+	//     X
+	//     Concept "soda drinker"
+	//   Inheritance
+	//     X
+	//     Concept "ugly"
```
Comment (ngeiswei) [inline tests/learning/xpattern-miner/c++-based/XPatternMinerUTest.cxxtest]: Please update that comment with some pattern that you expect.

**Comment by Bitseat** [comment]
sample mining result I've got.
```atom[982,depth=1]:
    (LambdaLink
      (VariableList
        (VariableNode "$X1") 
        (VariableNode "$X2") 
        (VariableNode "$P1") 
      ) 
      (AndLink
        (EvaluationLink
          (VariableNode "$P1") 
          (ListLink
            (VariableNode "$X1") 
            (VariableNode "$X2") 
          ) 
        ) 
        (EvaluationLink
          (PredicateNode "sumti1")
          (ListLink
            (VariableNode "$X1") 
            (ConceptNode "do") 
          ) 
        ) 
      ) 
    ) 

```

**Comment by linas** [comment]
@ngeiswei  should this be merged? This looks OK to me.

**Comment by linas** [comment]
Merged in 6cc7e6b6e2268792f5fd19763e95bc18d4fa7645 after resolving conflicts

**Pull request closed without merging**